Microsoft.Toolkit.Uwp.UI.Controls/Microsoft.Toolkit.Uwp.UI.Controls.csproj

```diff
@@ -4,45 +4,42 @@
+- Carousel: Presents items in a carousel control.
+- DataGrid: Control that presents data in a customizable table of rows and columns.
+- DockPanel: Defines an area where you can arrange child elements either horizontally or vertically, relative to each other.
+- DropShadowPanel: DropShadowPanel control allows the creation of a DropShadow for any Xaml FrameworkElement in markup.
+- Expander: Expander control allows user to show/hide content based on a boolean state.
+- GridSplitter: GridSplitter represents the control that redistributes space between columns or rows of a Grid control.
+- HeaderedContentControl: Allows content to be displayed with a specified header.
+- HeaderedItemsControl: Allows items to be displayed with a specified header.
+- HeaderedTextBlock: The HeaderedTextBlock control is designed to provide a header for read only text. This control is useful for displaying read only forms.
+- ImageCropper: ImageCropper control allows user to crop image freely.
+- ImageEx: Images are downloaded asynchronously showing a load indicator and can be cached.
+- InAppNotification: The In App Notification control offers the ability to show local notifications in your application.
+- InfiniteCanvas: InfiniteCanvas is a canvas that supports Infinite Scrolling, Ink, Text, Format Text, Zoom in/out, Redo, Undo, Export canvas data, Import canvas data.
+- LayoutTransformControl: Control that implements support for transformations as if applied by LayoutTransform.
+- Loading: LoadingControl helps to show content with animation to the user while the app is doing some calculation.
+- MarkdownTextBlock: An efficient and extensible control that can parse and render markdown.
+- MasterDetailsView: The MasterDetailsView control allows the user to implement the Master/Details design pattern.
+- OrbitView: The OrbitView Control positions items in a circle around a center element and supports orbits and anchors.
+- RadialGauge: The radial gauge displays a value within a range, using a needle on a circular face.
+- RadialProgressBar: The radial progress bar displays progress as a circle getting filled.
+- RangeSelector: The RangeSelector is a "double slider" control for range values.
+- RemoteDevicePicker: Remote Device Picker Control for Project Rome.
+- RotatorTile: RotatorTile is an ItemsControl that rotates through a set of items one-by-one.
+- ScrollHeader: A UI control that works as a ListView or GridView header control with quick return, sticky and fade behavior.
+- StaggeredPanel: The StaggeredPanel allows for layout of items in a column approach where an item will be added to whichever column has used the least amount of space.
+- TabView: A control for displaying multiple items in the same space and allows a user to easily switch between them.
+- TextToolbar: A Toolbar for Editing Text attached to a RichEditBox.
```

Comment:
_"It can format RTF and Markdown, or use a Custom Formatter, and specify your own Formatter with Buttons and Actions."_

We should add this forgotten explanation.

Reply:
We have a 4000 character limit for NuGet descriptions, so I think I removed it to not hit the limit.